### ISSUE TITLE
+exec console command doesnt work

### ISSUE THREAD

**Issue by hanm13** [Facepunch/garrysmod-issues#1748]
Hey I tried to load other server.cfg name
I created cfg name called c18.cfg
and I added to the run command +exec c18.cfg
and its loaded the regular server.cfg

**Comment by robotboy655** [comment]
Are we talking command line parameters for srcds?
It will load server.cfg regardless of what you try to exec with +exec, try adding ```echo testing``` into your c1.cfg and see if you can find "testing" in the console.

**Comment by hanm13** [comment]
Yes its command line
./srcds_run -console -game garrysmod +maxplayers 80 +exec c18.cfg +map rp_c18_v1.bsp -secure -port 26553

**Comment by hanm13** [comment]
the " testing" loaded on the console

**Comment by robotboy655** [comment]
So it is working properly.

**Comment by hanm13** [comment]
I added to the file other hostname and rcon but its keep loading the regular server.cfg ...

**Comment by Bo98** [comment]
+exec doesn't replace server.cfg, it's for loading additional configs.

**Comment by hanm13** [comment]
How I can replace the server.cfg ?

**Comment by robotboy655** [comment]
Empty it? Delete it?

**Comment by hanm13** [comment]
I want to run two servers from one directory but different server.cfg per server

**Comment by Bo98** [comment]
Empty server.cfg and create different cfgs to use in +exec?

For example:

server.cfg = empty
server1.cfg = server 1's config - use as +exec server1.cfg
server2.cfg = server 2's config - use as +exec server2.cfg

**Comment by hanm13** [comment]
Thank you !

**Comment by Acecool** [comment]
Actually; I've noticed this on the game-client AND server... Regardless of how I type it, +exec or -exec it won't execute the cfg and it must be done manually. I have a highres.cfg script for my main gmod window, and lowres.cfg for the smaller window when I create multiple clients to work on my dev-server.

I've tried launching from bat files, Steam, etc.. Client won't execute the cfg. For the server, it doesn't appear to execute either but I haven't confirmed it; the CLIENT not executing I have confirmed.
